Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift

```diff
@@ -156,7 +163,13 @@ class AssignmentListViewController: UIViewController, ColoredNavViewProtocol, Er
             self?.updateFilterButton()
             self?.selectFirstCellOnIpad()
             self?.loading = false
+            let maxPrefetch = 2
+            if self?.prefetchCount ?? 0 < maxPrefetch {
+                self?.prefetchCount +=  1
+                if self?.pagingCursor != nil { self?.fetchData() }
+            }
```

Comment:
Help me understand why it's prefetching two? Will it not fetch as needed? What happens when there is a 3rd cursor?

Reply:
The page size is 10 on the simulator 11 rows can fit without scrolling.  It only pages when scrolled in the method using the `UITableViewDataSourcePrefetching` in the ```func tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath])``` is called.